model/accounts_mgmt/v1/feature_toggle_resource.model

```diff
@@ -16,8 +16,9 @@ limitations under the License.
 
 // Manages feature toggles.
 resource FeatureToggle {
-	// Retrieves the details of the feature toggle.
-	method Get {
-		out Body FeatureToggle
+	// Retrieves the details of the feature toggle by providing query context
+	method Post {
+		in Request FeatureToggleQueryRequest
```

Comment:
This is not very restful. Why not add the Organization Id to the `FeatureToggle` type?

Reply:
Because the organization id is not related to the response.

The request need to contain some context. In this case - the org id. The response however, is true/false.